R/get_gsw.R

```diff
@@ -1,3 +1,90 @@
+#' Global Surface Water Change
```

Comment:
Could you please add the scientific reference of this data set? Also I would suggest to write a common paragraph (2 or 3 sentences) of the general background of the data (e.g. calculated by JRC based on ...) and repeat that for all resources before going into the details of that specific resource.